src/main/java/ch/njol/skript/expressions/ExprDateAgoLater.java

```diff
@@ -46,20 +46,24 @@
                 "%timespan% (later|(from|after) [the] [date] %-date%)");
     }
 
-    @SuppressWarnings("null")
+    @Nullable
     private Expression<Timespan> timespan;
-    @SuppressWarnings("null")
+    @Nullable
     private Expression<Date> date;
 
     private boolean ago;
 
-    @Nullable
+	@SuppressWarnings("null")
+	@Nullable
     @Override
     protected Date[] get(Event e) {
+		if (this.timespan == null || this.date == null) {
```

Comment:
this does not function the same as old behavior, the old behavior would replace a null `date` with the current date, while this implementation just returns null for a null `date`

Reply:
Will ``this.date`` ever be null? Like, the ``Expression<Date>`` variable itself, not its content.

Reply:
yes, in the second pattern the date expr is nullable

Reply:
Fixed in next commit